optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/test/resources/META-INF/beans.xml

```diff
@@ -14,12 +14,11 @@
   ~ limitations under the License.
   -->
 
-<beans>
-  <scan>
-    <exclude name="**.client.**"/>
```

Comment:
@ge0ffrey the errai-cdi refactoring has already happened. There's a errai-cdi-server, errai-cdi-shared and errai-cdi-client module (which won't be on the classpath of the server). So, there should be no unnecessary scanning going on anymore...at least not for errai cdi.